account_invoice_comment_template/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+column_renames = {
+    "account_move": [
+        ("comment_template1_id", "old_comment_template1_id"),
```

Comment:
Why are you renaming columns if the previous name is not overlapped by anything? The only reason for renaming columns is for avoiding to conflict with the same name, and as it's very unlikely that other module as this field name, it's better to save on scripts doing the renaming + avoid to use the renamed one on further scripts. Anyways, if needed, the usual technique is to rename to `None`, for letting OpenUpgrade to put the name with `openupgradelib.get_legacy_name`.